Adding usecases and requirements to Rucio (#565)
EGI-Federation/documentation: content/en/users/data/management/rucio/_index.md

allows communities to easily start using and/or test Rucio with little setup
cost.

## Requirements to consider to use Rucio

> For Rucio to mangage your data in this setup, Rucio will need X509 certificate

Reviewer: ```suggestion
For Rucio to manage your data in this setup, Rucio will need X509 certificate
```

Author: changes made

access, or soon, through
[EGI Check-in](../../../../providers/check-in/_index.md) to:

- Your experiments Storage Element that is X509 capable
- Your experiments Tape Archive that is X509 capable
- Your experiments Voms server information to check Users credentials against

Rucio is a system that sits on top of already established storage elements to

> unify users access for data management, and retrival. Rucio consists of a

Reviewer: ```suggestion
unify users access for data management, and retrieval. Rucio consists of a
```

Author: changes made

database of the storage element's details, users and their access credential

> information, and access levels, and the data and its location, not a direct data

Reviewer: ```suggestion
information, access levels and the data and its location. Rucio is  not a direct data
```

Author: changes made

storage solution.

## Rucio Use Cases

Rucio is a data management software, that integrates with your experiments
currently provisioned storage. Therefore, this section will deal with usecases
that you should consider Rucio for.

### Management of data for archive when no longer used actively

> One that may be useful is if your experiment has data that is newly generated,

Reviewer: not so much clear this paragraph

Author: rewrote in latest PR, hopefully makes more sense to you

that data will be accessed much more than older data. This new data is also
precious and needs to be preserved. To ensure the integrity of the data, Rucio
can be instructed by two rules, one to write the data to a cheap long term
archive of tape storage, the second to ensure the data is also available on HDDs
or, if available SSDs. Then as the usage of this data declines, the data on the
more accessible HDD or SSD can be removed, in the knowledge that if the data is
ever required again, Rucio can stage the data from tape back to disk.

### Management of data between sites for jobs

> Another useful usecase for Rucio is to mangage the data between different sites

Reviewer: ```suggestion
Another useful use case for Rucio is to manage the data between different sites
```

Author: changes made
